### PR TITLE
OSM importer: removal radius

### PR DESCRIPTION
**Pull request by fabienrohrer**
Documentation for feature https://github.com/omichel/webots/pull/4789

### CODE REVIEW

**Comment by fabienrohrer** [comment]
👍

File: automobile/openstreetmap-importer.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ You can use several arguments with this script:
 | --disable-multipolygon-buildings | Does not generate buildings from multipolygon                                                                              | By default, buildings are generated from multipolygon                     |
 | --projection           | Defines the projection parameters, the projection parameters should be defined following the PROJ.4 rules (https://trac.osgeo.org/proj/wiki/GenParms). Note that if you are not using the default projection, the GPS model of Webots may not match with the generated world. | By default, an empty string is used to define an UTM projection |
 | --extract-projection   | Extracts the projection from the OSM file, displays it and exits.                                                                    | By default, this parameter is disabled.                                   |
-
+| --removal-radius       | Specifies the radius (in meters) around each road waypoint outside which any object are removed.                                     | By default this feature is disabled (radius = 0.0).                       |
```
Comment (omichel) [inline automobile/openstreetmap-importer.md]: Either "all objects are" or "any object is".

**Comment by fabienrohrer** [comment]
👍

File: automobile/openstreetmap-importer.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ You can use several arguments with this script:
 | --disable-multipolygon-buildings | Does not generate buildings from multipolygon                                                                              | By default, buildings are generated from multipolygon                     |
 | --projection           | Defines the projection parameters, the projection parameters should be defined following the PROJ.4 rules (https://trac.osgeo.org/proj/wiki/GenParms). Note that if you are not using the default projection, the GPS model of Webots may not match with the generated world. | By default, an empty string is used to define an UTM projection |
 | --extract-projection   | Extracts the projection from the OSM file, displays it and exits.                                                                    | By default, this parameter is disabled.                                   |
-
+| --removal-radius       | Specifies the radius (in meters) around each road waypoint outside which any object is removed.                                      | By default this feature is disabled (radius = 0.0).                       |
```
Comment (omichel) [inline automobile/openstreetmap-importer.md]: I would use "beyond" instead of "outside".

**Pull request merged**